### ISSUE TITLE
Controls are not visible until I actually start the video.

### ISSUE THREAD

**Issue by mleyen** [ParticleCore/Particle#26]
When autoplay is disabled, only the red play button is visible until I start the video. It would be nice to have the full control bar before starting the video.

**Comment by ParticleCore** [comment]
This is the default YouTube behavior when you have the controls' option set to hide or if you are using the new transparent player.
